src/CovidDataDashboard/CovidDataStateTrend.tsx

```diff
@@ -0,0 +1,141 @@
+import React, { FunctionComponent, useEffect, useState } from "react"
+import { useTranslation } from "react-i18next"
+import { SvgXml } from "react-native-svg"
+import { StyleSheet, TouchableOpacity, View } from "react-native"
+import { useNavigation } from "@react-navigation/native"
+
+import { calculateCasesPercentageTrend } from "./covidData"
+import { CovidData } from "./covidDataAPI"
+import { HomeStackScreens, Stacks } from "../navigation"
+
+import { Text } from "../components"
+import { Icons } from "../assets"
+
+import { Colors, Typography, Spacing, Outlines, Iconography } from "../styles"
+
+type CovidDataStateTrendProps = {
+  lastWeekData: CovidData[]
+  stateAbbreviation: string
+}
+
+const CovidDataStateTrend: FunctionComponent<CovidDataStateTrendProps> = ({
+  lastWeekData,
+  stateAbbreviation,
+}) => {
+  const { t } = useTranslation()
+  const navigation = useNavigation()
+  const [casesPercentageTrend, setCasesPercentageTrend] = useState<
+    number | null
+  >(null)
+
+  const handleOnPressDataClip = () => {
+    navigation.navigate(Stacks.Modal, {
+      screen: HomeStackScreens.CovidDataDashboard,
+    })
+  }
+
+  useEffect(() => {
+    setCasesPercentageTrend(calculateCasesPercentageTrend(lastWeekData))
+  }, [lastWeekData])
+
+  if (casesPercentageTrend === null) {
+    return null
+  }
+
+  const trendPercentageValue = `${Math.abs(casesPercentageTrend)}%`
+
+  const trendInfo =
+    casesPercentageTrend > 0
+      ? {
+          text: t("covid_data.up_from_last_week"),
+          indicator: <SvgXml xml={Icons.ChevronUp} fill={Colors.black} />,
+        }
+      : {
+          text: t("covid_data.down_from_last_week"),
+          indicator: <SvgXml xml={Icons.ChevronDown} fill={Colors.black} />,
+        }
+
+  return (
+    <TouchableOpacity
```

Comment:
Thoughts on pulling the `TouchableOpacity` and text with the `style.sectionHeaderText` out into the `Home/index`? It may be more organized there because that's how all the other "modules" on that page are arranged.

Reply:
There is a component that is surrounding this, the data clip one that is making a number of checks on the data, to make sure that this component only gets displayed if the data is present, making it a bit simpler by splitting that complexity. I feel like moving it up because of the semantics and the positioning would move the complexity back up there. Thoughts?

Reply:
Ah I see. Yeah probably simpler this way, at least for now.